test/AspNet.Security.OAuth.Providers.Tests/Alipay/AlipayTests.cs

```diff
@@ -77,6 +77,47 @@ public async Task BuildChallengeUrl_Generates_Correct_Url(bool usePkce)
         }
     }
 
+    [Fact]
+    public async Task Cannot_Sign_In_Using_Alipay_With_Sub_Code_Error()
+    {
+        // Arrange
+        static void ConfigureServices(IServiceCollection services)
+        {
+            services.PostConfigureAll<AlipayAuthenticationOptions>((options) =>
+            {
+                options.ClientId = "error-id";
+            });
```

Comment:
This can be simplified to:

```suggestion
            services.PostConfigureAll<AlipayAuthenticationOptions>((options) => options.ClientId = "error-id");
```